Include/internal/pycore_compile.h

```diff
@@ -63,6 +64,115 @@ typedef struct {
     int u_firstlineno; /* the first lineno of the block */
 } _PyCompile_CodeUnitMetadata;
 
+struct compiler;
```

Comment:
Should these be Py-prefixed? I thought the rule now was to use a Py prefix even for internal symbols in headers.